randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -592,8 +598,11 @@ Extra - room_id: [14]
   * Extra - door_idx: (28,)
   > Door to Forbidden Entrance
       Any of the following:
+          # Wall jump to the top: https://youtu.be/D8a57K7PwZ0
           Space Jump or Wall Jump (Beginner)
+          # Freeze the Menus: https://youtu.be/_HmPed89A7E
           Stand On Frozen Enemies (Beginner) and Can Freeze Enemies With Any Weapon
```

Comment:
this should be bumped to advanced at the very least

Reply:
Is this referring to "Freeze the Menus" or the Wall jump?

Reply:
freezing